alphagov/paas-cf
# [#144443475] Simplify how we run tests

## What

Story: [Review how we run upstream acceptance tests](https://www.pivotaltracker.com/story/show/144443475)

* Stop faking errand environment to run acceptance and smoke tests and use the same approach as upstream
* Fix usage of temporary credentials
* Remove obsolete properties

## How to review

* Deploy and let all tests run, including continuous smoke tests
* Check `config.json` doesn't include the main admin credentials, it must be the temporary ones

## Who can review
Not me
